gdcdictionary/schemas/virus_sequence_blastn.yaml

```diff
@@ -4,13 +4,13 @@ id: "virus_sequence_blastn"
 title: Virus Sequence Blastn
 type: object
 namespace: https://chicagoland.pandemicresponsecommons.org
-category: analysis
+category: genomic_data_file
```

Comment:
`genomic_data_file` is the category we use for the "Genomics" tab in the explorer. So if we use it for this data, that's where it'll show up. Just mentioning it for Gina to think about

Reply:
After discussion, we agreed to make all related nodes as `genomic_data_file`